src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts

```diff
@@ -599,10 +611,10 @@ export class FormTransferTransactionTs extends FormTransactionBase {
      * Resetting the form when choosing a multisig signer and changing multisig signer
      * Is necessary to make the mosaic inputs reactive
      */
-    @Watch('selectedSigner')
```

Comment:
Have we tested the case when the user changes the current account from the top menu? Is the form resetting correctly?

Reply:
Currently, it got cleared -> https://share.getcloudapp.com/kpunrZQj

Reply:
The upper menu switch accounts action should be working properly after the last change.

Reply:
@bassemmagdy 
What is expected behaviour after account switch in the top menu?
A) Clear form and reload mosaics
B) Keep form (recipient, message, fee) and reload mosaics

Reply:
clear the form and reload mosaics